documentation/nginx_site.md

```diff
@@ -0,0 +1,35 @@
+# nginx_site
+
+[Back to resource list](../README.md#resources)
+
+## Actions
+
+- `:create` - Create an nginx site configuration file
+- `:delete` - Delete an nginx site configuration file
+
+## Properties
+
+| Name                   | Type          | Default                          | Description                                                         | Allowed Values      |
```

Comment:
Is there any reason why we need the `Allowed Values` column if we're not using it?